src/main/java/creativei/dao/impl/InquiryCustomDaoImpl.java

```diff
@@ -60,6 +61,20 @@ public Inquiry findOne(Long aLong) {
         return inquiryDao.findOne(aLong);
     }
 
+    @Override
+    public Integer findCountByInquiryDate(Date inquiryDateTo, Date inquiryDateFrom) {
+        return inquiryDao.findCountByInquiryDate(inquiryDateTo,inquiryDateFrom);
+    }
+
+    @Override
```

Comment:
What are 2 different case index values?

Reply:
caseIndex 3 "Likely" and caseIndex 4 "HotLead"